src/reducers/SelectedRevisionsSlice.ts

```diff
@@ -23,26 +24,37 @@ const selectedRevisions = createSlice({
       state.base = [action.payload.selectedRevisions[0]];
       //returns array without first element
       state.new = action.payload.selectedRevisions.slice(1);
+      state.editModeRevisions = action.payload.selectedRevisions;
     },
-    deleteRevision(
+    saveUpdatedRevisions(
+      state,
+      action: PayloadAction<{
+        selectedRevisions: RevisionsList[];
+      }>,
+    ) {
+      state.base = [action.payload.selectedRevisions[0]];
+      //returns array without first element
+      state.new = action.payload.selectedRevisions.slice(1);
+      state.editModeRevisions = action.payload.selectedRevisions;
+    },
+    updateEditModeRevisions(
       state,
       action: PayloadAction<{
         selectedRevisions: RevisionsList[];
         isBaseDeletion: boolean;
+        isAddChecked: boolean;
```

Comment:
having comments explaining what these 2 booleans are for would be useful.

My understanding:
* isBaseDeletion => the user deleted the base rev
* isAddChecked => I honestly don't know :D